andela/mag-ah-frontend
# #159965257 set up a react app for Authors Haven

#### What does this pull request do
- sets basic configuration and boilerplate code for the new app

#### Description of tasks to be completed
- ensure boilerplate code for new react app is ready
- adding Eslint configuration with Airbnb option for the app

#### How can this be tested manually
- Using a terminal, clone [this](https://github.com/andela/mag-ah-frontend.git) repository
- Execute `cd mag-ah-frontend`
- Check out to `ch-app-set-up-159965257`
- install [node with npm](https://www.npmjs.com/get-npm)
- execute `npm install` to install app dependencies
- start the app by executing `npm start`; it will open a browser window pointing to the app. If it does not open, point your browser to `http://localhost:3000/`

#### What are the pivotal tracker story for the task
[set up a react app for the project](https://www.pivotaltracker.com/story/show/159965257)
#### screenshot
![image](https://user-images.githubusercontent.com/9263906/46353391-bcb7ee00-c664-11e8-841c-164521211823.png)
